git.yazi/main.lua

```diff
@@ -193,19 +186,19 @@ local function fetch(_, job)
 	local changed, ignored = {}, {}
 	for line in output.stdout:gmatch("[^\r\n]+") do
 		local sign, path = match(line)
-		if sign == CODES.excluded then
-			ignored[path] = sign
-		else
-			changed[path] = sign
+		if sign and path then
+			if sign == CODES.ignored then
```

Comment:
Only ignored *directories* need to `propagate_down` - why have you also included ignored *files* here? Is this necessary?